docs/howto/openstack/nova/resize-server.md

```diff
@@ -0,0 +1,125 @@
+# Resizing a server
+
+This guide will walk you through the required steps to change the number of CPU cores and the amount of memory your server has access to, this is done my changing the server's [flavor](/reference/flavors/).
+
+[Resize](https://docs.openstack.org/nova/latest/admin/configuration/resize.html) (or Server resize) is the ability to change the flavor of a server, thus allowing it to upscale or downscale according to user needs. A resize operation is a two-step process for the user: the first step is to perform the resize, and the second step is to either confirm (verify) success and release the old server or to declare a revert to release the new server and restart the old one.
+
+## Prerequisites
+
+You need to have a server you wish to resize. Additionally, if you prefer to work with the OpenStack CLI, then make sure to properly [enable it first](/howto/getting-started/enable-openstack-cli/).
+
+## Listing available flavors
+
+=== "{{gui}}"
+    Navigate to the server list.
+
+    ![navigation-panel](assets/resize-server/01-left-side-panel.png)
```

Comment:
Please add a proper [alt text](https://accessibility.huit.harvard.edu/describe-content-images) here (this comment applies to all inline images in this PR).

Reply:
I'm not certain what you are asking me do, am I totally wrong with this example?
```
    ![The left hand side navigation panel with the word, servers, highlighted.](assets/resize-server/01-left-side-panel.png)
```

Reply:
Not totally wrong at all. :) As a minor improvement I'd put:

```markdown
![The left hand side navigation panel, with the word "Servers" highlighted.](assets/resize-server/01-left-side-panel.png)